calico-vpp-agent/services/service_server.go

```diff
@@ -158,7 +159,7 @@ func (s *Server) resolveLocalServiceFromService(service *v1.Service) *LocalServi
 	return s.GetLocalService(service, ep)
 }
 
-func (s *Server) resolveLocalServiceFromEndpoints(ep *v1.Endpoints) *LocalService {
+func (s *Server) resolveLocalServiceFromOldEndpointSlice(ep *discoveryv1.EndpointSlice) *LocalService {
```

Comment:
Do we really need to differentiate code paths for old and new endpointslices ?

Reply:
Not really, we can use a flag..